server/models/index.js

```diff
@@ -33,42 +33,33 @@ let saveProduct = (productId, productItem, pictureUrl) => {
   });
 };
 
-let saveWishlist = (products, username) => {
+let saveWishlist = async (products, username) => {
   const instance = new MyWishlistModel({
     products: products,
     username: username
   });
-  instance.save((err) => {
-    if (!err) {
-      console.log('success');
-    }
-  });
+  return await instance.save();
```

Comment:
When you use async await normally you would want to do somthing like
`try {
  await action()
catch (err) {
  doSomething(err)
}`
Other wise if your async action fails you will get errors that may crash your server

Reply:
I don't know why line breaks didn't show up in this. 
try {
  await action();
} catch (err) {
  doSomething(err)
}

Reply:
I am not sure about this but I will keep it in mind.